### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59460]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
When printing messages in a conversation, you're unable to successfully print when using a keyboard only. Selecting the either radio button using keyboard and tabbing to the Print button] results in an error, while clicking on a radio button and clicking Print does not.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Log in with a test user and navigate to a message thread.
2. Press [tab] until keyboard focus moves to the Print button.
3. Press [enter] to open the printing modal.
4. Press [tab] to move focus to the first radio button.
5. Press [spacebar] to select the "Print only this message" radio button. Note that the radio button appears to be selected and the `va-radio-option` component has a `checked` attribute.
6. Press [tab] to move focus to the Print button.
7. Press [enter] to print.
8. Note that you receive an error message, "Please select an option to print."
9. Refresh the page.
10. Repeat steps (1) through (4).
11. Press [down arrow] to select the "Print all messages in this conversation" radio button. Note that the radio button appears to be selected and the `va-radio-option` component has a `checked` attribute.
12. Press [tab] to move focus to the Print button.
13. Press [enter] to print.
14. Note that you receive an error message, "Please select an option to print."

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

### Platform Recommendation
I'm guessing whatever is validating the selection here is listening for a click event, rather than checking to see if one of the options is checked?

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by bryan-riley-va** [comment]
JIRA ticket created: https://jira.devops.va.gov/browse/MHV-45988

**Comment by oleksii-morgun-va** [comment]
@AngelaFowler82 this is now in staging ready for testing

**Comment by AngelaFowler82** [comment]
I can now verify that the print function is now keyboard accessible. Closing as successfully resolved.
